<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1089</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="apiKeyInput">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="iterationsInput">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="clustersInput">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>API Key</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of Clusters</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of Iterations</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fetchDataButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>100</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Fetch Data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="runClusteringButton">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>450</y>
      <width>271</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Run Clustering</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="neoTable">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>361</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="clusterTable">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>361</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="statusOutput">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>190</y>
      <width>241</width>
      <height>74</height>
     </rect>
    </property>
   </widget>
   <widget class="QChartView" name="chartView" native="true">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>591</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1089</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header location="global">QtCharts/QChartView</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
